c2corg_ui/templates/route/helpers/view.html

```diff
@@ -14,7 +14,22 @@
       </div>
       <span class="detail-text accordion" id="detail-orientation">
         <span class="areas">${show_areas(route)}</span>
+      </span>
+    </div>
+  % endif
+</%def>
 
+## ORIENTATION
+<%def name="get_route_orientation(route)">\
+  % if route.get('orientations'):
+  <div class="name-icon-value"
+       ng-click="detailsCtrl.openTab($event)">
+      <h4><span translate>Orientation</span><span class="glyphicon glyphicon-menu-right"></span></h4>
```

Comment:
Please use the usual attribute name, not a new one (that will require a new translation) => ``orientations`` (ditto 3 line below).